src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -316,6 +316,14 @@ class TableConstraintSegment(BaseSegment):
         name="declared_variable",
         type="variable",
     ),
+    BooleanDynamicSystemVariablesGrammar=OneOf(
+        # Boolean dynamic system varaiables can be set to ON/OFF, TRUE/FALSE, or 0/1:
```

Comment:
0/1 is not covered here. Guessing not needed in this Grammar but should we at least add a test case to show this works anyway?

Reply:
yeah this grammar is just to get these parsing as keywords and therefore have the correct capitalisation. Best to let 0/1 parse in the same way as the rest of the integers 👍 

I've added an extra unit test